.github/workflows/bindings_nodejs.yml

```diff
@@ -279,6 +279,13 @@ jobs:
         working-directory: .
         run: cp LICENSE NOTICE ./bindings/nodejs
 
+      - name: Publish Dry Run
+#        if: "startsWith(github.ref, 'refs/tags/') && contains(github.ref, '-')"
+        # Since this command will not exit with non-zero code when file missing,
```

Comment:
We can build all files.

Reply:
I was doing that just now. Pretty good, found a problem.